distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1204,6 +1204,92 @@ public void testNoEnforceValidityErrorsOrWarnings() throws Exception {
     }
 
     @Test
+    public void testQueryErrors() throws Exception {
+        getServiceManager().startFeature(true,
+                "catalog-plugin-metacard-validation",
```

Comment:
I don't know how performant it is to try to start a feature when one is already started. I would feel most comfortable with taking it out - if another test stops the feature, it's that test's responsibility to restart it.